Makefile

```diff
@@ -5,7 +5,7 @@ node_modules/@financial-times/n-gage/index.mk:
 -include node_modules/@financial-times/n-gage/index.mk
 
 unit-test:
-	jest src/*
+	# skip
```

Comment:
do we have no tests for this?

Reply:
We do, and they fail right now, I guess because we changed react from v0.14 to v15. - there were some substantial changes in ReactTestUtils. Created separate card in Tech Hygiene for that.